services/storage/object/taxonomy.md

```diff
@@ -0,0 +1,28 @@
+# Object Storage Taxonomy
+
+This _service-level taxonomy_ documents the minimual set of features
+that should be present for a service to be considered portable for
+use in financial services ecosystems.
+
+## Taxonomy
+
+| Taxonomy ID | Feature | Description |
+| ----------- | ------- | ----------- |
+| CCC-020101 | Buckets | Concept of having uniquely identifiable containers or buckets to store objects. |
+| CCC-020102 | Metadata | Support storing, accessing, and managing of object metadata for stored objects. |
+| CCC-020103 | Scalability - Capacity Limit | Ability to store unlimited number of objects under a given maximum total capacity. |
+| CCC-020104 | Scalability - Object Size Limit | Ability to store large objects under a given maximum object size. |
+| CCC-020105 | Durability | High durability for stored objects through redundancy and replication. |
+| CCC-020106 | Availability | High availability for stored objects through replication over multiple availability zones within a region. |
+| CCC-020107 | Performance - Transaction Rate Limits | High throughput and low latency for read/write operations under a given maximum transaction rate limits.  |
```

Comment:
This doesn't read quite right to me. Maybe it could be changed to:

High throughput and low latency for read/write operations under given maximum transaction rate limits.

or

High throughput and low latency for read/write operations under a given maximum transaction rate limit.

Reply:
Thanks! selected the first version as there are 2 rates, max read limit and max write limit